jsonrpc/jsonrpc.go

```diff
@@ -2,5 +2,5 @@ package jsonrpc
 
 const (
 	// ChainID represents the L2 chainID
-	ChainID = uint64(1000)
+	ChainID = uint64(1000) //nolint:gomnd
```

Comment:
```suggestion
	ChainID uint64 = 1000
```

Reply:
I think this should come from the network configuration instead of being a constant

Reply:
I converted this to be a constant as it is going to be the same no matter the Testnet we are running.
I recommend keeping this way until we realize how we are going to handle the ChainID for the new version.

Reply:
Hm, this is out of scope (I had to add the `nolint:gomnd` for the code to pass the linter). I'll open a new PR with the fix.